clrdefinitions.cmake

```diff
@@ -15,6 +15,7 @@ if (CLR_CMAKE_TARGET_ARCH_AMD64)
   if (CLR_CMAKE_PLATFORM_UNIX)
     add_definitions(-DDBG_TARGET_AMD64_UNIX)
   endif()
+  add_definitions(-DBIT64=1)
   add_definitions(-D_TARGET_AMD64_=1)
```

Comment:
128 bit processors are not going mainstream anytime soon. And in the unlikely case they will ever become mainstream, the 32-bit ones are likely to be obsolete by then.